tests/System.IO.Pipelines.Tests/SocketsFacts.cs

```diff
@@ -176,6 +175,9 @@ public void CanCreateWorkingEchoServer_PipelineSocketServer_NonPipelineClient()
 
         static async Task<Tuple<int, int, int>> PingClient(IPipeConnection connection, int messagesToSend)
         {
+            Span<byte> _ping = new Span<byte>(Encoding.ASCII.GetBytes("PING"));
```

Comment:
This is almost certainly being caused by the WriteAsync extension method on IPipeWriter (WriteAsync). Async methods cannot take Span<T> as an argument.